wp-content/themes/theme_name/inc/enqueue.php

```diff
@@ -42,7 +32,6 @@ function register_scripts() {
 /**
  * Remove Contact form 7 Includes
  */
```

Comment:
We should have a check to see if wpcf7 is even installed before doing any kind of removal or modification of the plugin scripts/styles (to avoid errors).